scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -624,6 +665,7 @@ function Install-Services {
 
 function Uninstall-Services {
     if (Get-Service $EdgeServiceName -ErrorAction SilentlyContinue) {
+        Set-Service -StartupType Disabled $EdgeServiceName -ErrorAction SilentlyContinue
         Stop-Service -NoWait -ErrorAction SilentlyContinue -ErrorVariable cmdErr $EdgeServiceName
```

Comment:
I think we need to review how we're uninstalling the service (E.g., Do we still need to use `-NoWait`? Can we get rid of the sleep?). But let's do that separately...